components/PostAttachment.php

```diff
@@ -84,12 +85,64 @@ public function set_calculated_image_size_cache( $image, $attachment_id, $size,
 		return $image;
 	}
 
+	/**
+	 * Filters 'img' elements in post content to add 'srcset' and 'sizes' attributes.
+	 *
+	 * @since 4.4.0
+	 *
+	 * @see   wp_image_add_srcset_and_sizes()
+	 *
+	 * @param string $content The raw post content to be filtered.
+	 *
+	 * @return string Converted content with 'srcset' and 'sizes' attributes added to images.
+	 */
+	public function make_content_images_responsive( $content ) {
```

Comment:
WP already generate srcset and sizes for <img> tag, so if you added this filter on the_content - then we should remove some WP default filter to prevent running some code which we don't need at all.